appdaemontestframework/appdaemon_mock/sccheduler.py

```diff
@@ -0,0 +1,148 @@
+
+import datetime
+import uuid
+import pytz
+from appdaemontestframework.appdaemon_mock.appdaemon import AppDaemon
+
+class Scheduler:
+    """Implement the AppDaemon Scheduler appropriate for testing and provide extra interfaces for adjusting the simulation"""
+    def __init__(self, AD: AppDaemon):
+        self.AD = AD
+        self._registered_callbacks = []
+
+        # Default to Jan 1st, 2000 12:00AM
+        # internal time is stored as a naive datetime in UTC
+        self.sim_set_start_time(datetime.datetime(2000, 1, 1, 0, 0))
+
+    ### Implement the AppDaemon APIs for Scheduler
+    async def get_now(self):
+        """Return current localized naive datetime"""
+        return pytz.utc.localize(self._now)
```

Comment:
This was some copy-paste from app daemon itself. I think this would make a difference if we were support async tests, but for now I don't see why this wouldn't work.

Reply:
Oh yeah. I noticed that after commenting. But well, what do we know, is it really copy when we improve upon the original ? 😁
Thx for updating